src/main/java/de/hpi/bpt/argos/storage/dataModel/entity/VirtualRoot.java

```diff
@@ -0,0 +1,64 @@
+package de.hpi.bpt.argos.storage.dataModel.entity;
+
+/**
+ * This is the virtual root node for the entity hierarchy.
+ */
+public class VirtualRoot extends EntityImpl {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this class "final" or add a public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS2974)